default.nix

```diff
@@ -1,40 +1,63 @@
-{ pkgs ? import <nixpkgs>{} } :
+{ pkgs ? import <nixpkgs>{}, versionArg ? "" } :
 let
   buildPrybar = attrs: pkgs.callPackage(import ./build-prybar.nix attrs) {};
+
+  julia = import ./languages/julia/julia.nix { inherit pkgs; };
```

Comment:
Do you think we will be able to depend on julia from `nixpkgs` at some point?

Reply:
We might have to step in and fix it in the short term. My hope for prybar is that this whole "Compile with C" thing can go away and we can start bringing the latest version without any worries.  The main reason why I can't use the official one yet is that the 1.5.4 version is identified as broken by Nix and the 1.6.1 has scattered the source files so compiling it in CGO is almost impossible :disappointed:

Reply:
Oof, yeah it might be time to help out in `nixpkgs`... Ideally we can get 1.6 going, the Julia folks seem to always want the latest version (very reasonable). Maybe we can chat with them and see if there's a new approach we need to take.